templates/addons/ddb/cf.yml

```diff
@@ -13,7 +13,6 @@ Resources:
     Metadata:
       'aws:copilot:description': 'An Amazon DynamoDB table for {{.Name}}'
     Type: AWS::DynamoDB::Table
-    DeletionPolicy: Retain
```

Comment:
If we are not using `storage init` to create an addons template. How do we make sure the template generated by `storage init` actually works 🤔

Reply:
oh nvm. i thought it was in e2e tests

Reply:
discussed offline!